src/lake/tests/manifest/lake-manifest-v1.0.0.json

```diff
@@ -0,0 +1,20 @@
+{"version": "1.0.0",
```

Comment:
Syntactic comment, but for somebody just looking at the manifest, this field is easy to confuse with the package version. But I suppose that was already the case while using sequential versioning.

Reply:
True, that is certainly something to keep in mind. I don't think I have good solution for that now in this PR, though.